src/app/FakeLib/PaketTemplateHelper.fs

```diff
@@ -254,21 +254,23 @@ module internal Rendering =
 ///
 /// ## Sample usage
 ///
-///     Target "Create Paket Template" (fun _ ->
-///        "./temp/paket.template"
-///        |> PaketTemplate (fun p -> { p with
-///                                     TemplateFilePath = Some "./deploytemp/paket.template"
-///                                     Id = Some "My.New.Package"
-///                                     Version = Some "0.0.1-alpha"
-///                                     Authors = ["Me"; "You"; "Someone Other"] })
-///                                     Files = [ Include ("./**/*.dll", "/lib/dlls")
-///                                               Exclude "./Foo/bar.dll"
-///                                               Include ("./*.jpg", "/images") ]
-///                                     Dependencies = [ "Paket1.FOO", AnyVersion
-///                                                      "Paket2.BAR", GreaterOrEqual CURRENTVERSION
-///                                                      "Paket3.BAZ", GreaterOrEqualSafe LOCKEDVERSION
-///                                                      "Paket4.BOO", GreaterOrEqual (Version "1.2.3") ]
-///     )
+///Target "Create Paket Template" (fun _ ->
```

Comment:
I pasted from a working script



 

From: Steffen Forkmann [mailto:notifications@github.com] 
Sent: 7. mars 2016 15:05
To: fsharp/FAKE <FAKE@noreply.github.com>
Cc: Alexey Zimarev <alex@zimarev.com>
Subject: Re: [FAKE] Fix sample usage mentioned in #1168 (#1169)

 

In src/app/FakeLib/PaketTemplateHelper.fs <https://github.com/fsharp/FAKE/pull/1169#discussion_r55206777> :

> -///     Target "Create Paket Template" (fun _ ->
> -///        "./temp/paket.template"
> -///        |> PaketTemplate (fun p -> { p with
> -///                                     TemplateFilePath = Some "./deploytemp/paket.template"
> -///                                     Id = Some "My.New.Package"
> -///                                     Version = Some "0.0.1-alpha"
> -///                                     Authors = ["Me"; "You"; "Someone Other"] })
> -///                                     Files = [ Include ("./**/*.dll", "/lib/dlls")
> -///                                               Exclude "./Foo/bar.dll"
> -///                                               Include ("./*.jpg", "/images") ]
> -///                                     Dependencies = [ "Paket1.FOO", AnyVersion
> -///                                                      "Paket2.BAR", GreaterOrEqual CURRENTVERSION
> -///                                                      "Paket3.BAZ", GreaterOrEqualSafe LOCKEDVERSION
> -///                                                      "Paket4.BOO", GreaterOrEqual (Version "1.2.3") ]
> -///     )
> +///Target "Create Paket Template" (fun _ ->

I don't think the indentation is right here

—
Reply to this email directly or view it on GitHub <https://github.com/fsharp/FAKE/pull/1169/files#r55206777> .  <https://github.com/notifications/beacon/ACsMVcQsRuOuw2prCQbYwE5TP-g3fYyuks5prDCfgaJpZM4HqzMm.gif>